### PR TITLE
add server version and other info to health endpoint

### LINKED ISSUE #48: Add server version and other info to health endpoint
The health endpoint should include some diagnostic info, in addition to the status code. At a minimum, this should include the server version, but other info is fair game

### PR DESCRIPTION
**Pull request by kolharsam**
When merged, this PR will:

- add `ldFlag` named `LeifDBVersion` on main.go which will be used within the `/health` endpoint as part of the response.
- updates the `Makefile` to include the `ldflag` argument while making builds.

Closes #48.

cc: @btmorr

### CODE REVIEW

**Comment by kolharsam** [comment]
@btmorr What other information would you like to include within the response of the `health` endpoint?

File: main.go [hunk 1]
```diff
@@ -28,6 +28,11 @@ var (
 	ErrInvalidTimeouts = errors.New("appendInterval must be shorter than minimum election window")
 )
 
+const (
```
Comment (btmorr) [inline main.go:31]: This variable does not get correctly overwritten if the variable is a const. Switch this to a var and it works as expected

**Comment by btmorr** [comment]
Oh, also--do commit the changes to the "docs" directory. Since the fields on one of the endpoints changed, this modifies the auto-generated swagger schema, which is what's in that directory

File: main.go (hunk 1 above)
Comment (kolharsam) [reply main.go:31]: Oh, snap! I think I missed this while refactoring it. Sorry.

**Pull request merged**